src/_common/screen/screen-service.ts

```diff
@@ -224,7 +223,8 @@ if (!GJ_IS_SSR) {
 	 * This is used internally to check things every time window resizes.
 	 * We debounce this and afterwards fire the resizeChanges for everyone else.
 	 */
-	fromEvent(window, 'resize')
-		.debounceTime(250)
-		.subscribe(() => Screen._onResize());
+	window.addEventListener(
```

Comment:
We might want to put the debounce logic within Subscription itself so that we can debounce ealier (before it even hits the event bus). The reason is that for stuff like resize, it gets called anytime the window resizes any 1px. We want to make sure it stays super fast if possible.

Reply:
this is mostly for backwards compat. if we need to react on resize quickly, wouldn't we normally use the resize observer instead?

Reply:
I think we need to discuss what I mean with this

Reply:
I was wrong